recipes/ropwr/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ropwr" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ropwr-{{ version }}.tar.gz
+  sha256: 723981924e57d0601a753a25d3d5d925da7c43d169d55f400cb00970b73e3605
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pytest
+    - pip
+  run:
+    - python >=3.7
+    - cvxpy >=1.1.14
+    - numpy >=1.16
+    - scikit-learn >=0.22
+    - scipy >=1.6.1
+    - pybind11
```

Comment:
I'm unclear why you need `pybind11` here as it's not a requirement of the package and isn't mentioned anywhere in the source. It **is** a build requirement of `cvxpy` (see https://github.com/conda-forge/cvxpy-feedstock/blob/main/recipe/meta.yaml), but isn't a runtime requirment so it's not included in the environment. I couldn't see any of the older failures in the logs, so if you could let me know why you think you need it, that would be good.

Reply:
For some reason, it didnt pass the checks when i didnt have pybind11 in. I could change it again to show you the problem.

Reply:
Would you mind removing it so I can see the error?

Reply:
Of course. Done.